src/App/ViewModels/Settings/SplitTunnelingContainerViewModel.cs

```diff
@@ -29,7 +29,7 @@ public bool IsSplitTunnelingOn
 
         public ISDK SDK { get; }
 
-        public bool CanAddItems => !IsSplitTunnelingOn && !SDK.IsConnected;
+        public bool CanAddItems => IsSplitTunnelingOn;
```

Comment:
Is it allowed to add items when connected to VPN server?

Reply:
Did a change and it is not allowed to add items when connected.